warden/x/warden/types/v1beta3/keychain.go

```diff
@@ -68,3 +80,15 @@ func (kf *KeychainFees) EnsureValid() error {
 
 	return nil
 }
+
+func NewKeybaseId(value string) (*KeybaseId, error) {
+	if value == "" {
+		return nil, nil
+	}
+
+	if len(value) != 16 {
+		return nil, fmt.Errorf("keybase id must be length of 16")
+	}
+
+	return &KeybaseId{Value: value}, nil
+}
```

Comment:
i'd keep it as "nil" since it's an optional field - but maybe we can add checks for empty strings at the above level instead of simply returning nil here

i'll let @backsapc decide